src/Helpers/constants.js

```diff
@@ -47,6 +47,10 @@ export const PATHS = {
         title: 'CVE',
         to: 'cves/:cve'
     },
+    cveDetailsEdgePage: {
+        title: 'CVE',
+        to: 'cves/:cve/manage-edge-inventory'
```

Comment:
The URL is quite long, wouldn't just `/edge` be sufficient?

Reply:
The edge team has introduced this URL pathname and did not want to interfere with their work. I will discuss your suggestion with them definitely.